start/src/main/java/org/apache/accumulo/start/classloader/vfs/UniqueFileReplicator.java

```diff
@@ -63,7 +63,12 @@ public File replicateFile(FileObject srcFile, FileSelector selector) throws File
 
     try {
       String safeBasename = UriParser.encode(baseName, TMP_RESERVED_CHARS).replace('%', '_');
-      File file = File.createTempFile("vfsr_", "_" + safeBasename, tempDir);
+      File file = null;
+      try {
```

Comment:
It's possible that the tempDir has been deleted and is causing this action to fail.
This code could check to see if the tempDir exists and throw a different exception type or log an error that the directory has been deleted.